plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/panels/managers/CodeModernizerBottomWindowPanelManager.kt

```diff
@@ -123,6 +133,7 @@ class CodeModernizerBottomWindowPanelManager(private val project: Project) : JPa
     fun setJobRunningUI() = setUI {
         add(BorderLayout.CENTER, buildProgressSplitterPanelManager)
         banner.updateContent(message("codemodernizer.toolwindow.banner.job_is_running"), AllIcons.General.BalloonInformation)
+        updateJobId()
```

Comment:
Make job ID more visible in Transformation Hub by putting it at the top next to the job run time.